### PR TITLE
Avatar edit

### PR DESCRIPTION
**Pull request by brandtkstrom**
Show avatar (user.imgUrl) on user and user edit page. Allow avatar updates by uploading image file. By no means a great solution, but it seems to work.

### CODE REVIEW

File: client/src/Elements/Components/AvatarUpdater.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+    const formData = new FormData();
+    formData.append('file', file);
+    formData.append('filename', file.name);
+
+    const token: string | null = localStorage.getItem('loginToken');
+    if (token == null) {
+      // No auth token
+      return;
+    }
```
Comment (Munch-Z) [inline client/src/Elements/Components/AvatarUpdater.tsx]: Do we want to throw up some errors along with these two return statements?

File: client/src/Elements/Components/AvatarUpdater.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      });
+
+      const userData = await response.json();
+
+      if (response.ok) {
+        localStorage.setItem('userInfo', JSON.stringify(userData));
+      }
+    } catch (error) {
+      // Todo - handle upload error
+    } finally {
```
Comment (Munch-Z) [inline client/src/Elements/Components/AvatarUpdater.tsx]: Add handling for this?

File: client/src/Elements/Components/AvatarUpdater.tsx (hunk 1 above)
Comment (brandtkstrom) [reply client/src/Elements/Components/AvatarUpdater.tsx]: This is a corner case that should never happen. If we are on the user edit screen, we should always have a token. If not, we really should kick the user to the home page.

File: client/src/Elements/Components/AvatarUpdater.tsx (hunk 2 above)
Comment (brandtkstrom) [reply client/src/Elements/Components/AvatarUpdater.tsx]: Are we using modals? Else I can just throw up an alert

Comment (Munch-Z) [reply client/src/Elements/Components/AvatarUpdater.tsx]: Not that I'm aware of. Alert should work for now, if we decided to modal it out later it's an easy change.

File: client/src/Elements/Components/AvatarUpdater.tsx (hunk 1 above)
Comment (Munch-Z) [reply client/src/Elements/Components/AvatarUpdater.tsx]: What about the file === null one?

Comment (brandtkstrom) [reply client/src/Elements/Components/AvatarUpdater.tsx]: Could, but that really means no file is selected. Modal or alert can be used to notify, but it's another corner case that shouldn't happen. I can add alerts for now.

**Pull request merged**